dns/tokenizer.py

```diff
@@ -427,6 +439,8 @@ def next(self):
             raise StopIteration
         return token
 
+    __next__ = next
```

Comment:
I'm not sure what you mean. It's just to comply with PEP3114 which changed iterator api from next to __next__, this will provide both, but who cares?